.gitignore

```diff
@@ -42,6 +42,7 @@ install_manifest.txt
 *.xcworkspace
 
 # additional *nix generated files
+/ch
```

Comment:
To be more general, we could ignore that file in any directory it happens to appear by simply removing the leading `/`